src/client/visualizations/table/table.tsx

```diff
@@ -243,34 +332,64 @@ export class Table extends BaseVisualization<TableState> {
       className: 'sort-arrow ' + commonSort.direction
     }) : null;
 
-    var cornerSortArrow: JSX.Element = null;
-    if (commonSortName === splitDimension.name) {
-      cornerSortArrow = sortArrowIcon;
-    }
+    var measureWidth = this.getIdealMeasureWidth(this.state.bodyWidth, essence);
 
-    var measuresArray = essence.getEffectiveMeasures().toArray();
+    return essence.getEffectiveMeasures().toArray().map((measure, i) => {
+      let amISorted = commonSortName === measure.name;
+      let classNames = [
+        'measure-name',
+        measure === hoverMeasure ? ' hover' : undefined,
+        amISorted ? 'sorted' : undefined
+      ].filter(Boolean);
 
-    var headerColumns = measuresArray.map((measure, i) => {
       return <div
-        className={'measure-name' + (measure === hoverMeasure ? ' hover' : '')}
+        className={classNames.join(' ')}
```

Comment:
you should use the `classNames` function as `classNames('measure-name', { hover: measure === hoverMeasure, sorted: amISorted })`

`import { ..., classNames } from '../../utils/dom/dom';`